scripts/build-infuser.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Seems like the script should document that the directory `/opt/tea.xyz/var/infuser` needs three checkouts in it.